var/spack/repos/builtin/packages/adios2/package.py

```diff
@@ -81,7 +83,7 @@ class Adios2(CMakePackage):
     conflicts('+shared ~pic')
 
     # DataMan needs dlopen
-    conflicts('+dataman', when='~shared')
+    conflicts('+dataman', when='+shared')
```

Comment:
Changed by mistake? (see comment)
```suggestion
    conflicts('+dataman', when='~shared')
```

Reply:
I'm not even sure why that's there. There's no inherent issue with using the dataman engine with either static or shared.  I'm digging into it and the sdk failure